lib/streamlit/elements/write.py

```diff
@@ -39,10 +40,12 @@
     types.ModuleType,
 )
 
+LOGGER = get_logger(__name__)
```

Comment:
nit: I think we started using `_LOGGER` to mark it as module internal. However, this is still very inconsistent across the codebase.